batcher/aligned-batcher/src/connection.rs

```diff
@@ -33,17 +35,7 @@ pub(crate) async fn send_batch_inclusion_data_responses(
             return Err(BatcherError::WsSinkEmpty);
         };
 
-        let sending_result = ws_sink
-            .write()
-            .await
-            .send(Message::binary(serialized_response))
-            .await;
-
-        match sending_result {
-            Err(Error::AlreadyClosed) => (),
-            Err(e) => error!("Error while sending batch inclusion data response: {}", e),
-            Ok(_) => (),
-        }
+        send_response(ws_sink, serialized_response).await;
```

Comment:
I would return a `Result` in `send_response` and show the log accordingly if it was successfully sent or not.

Reply:
Discussed offline.